packages/lesmis/src/components/Report.js

```diff
@@ -80,7 +81,7 @@ export const Report = React.memo(
   }) => {
     const { code: itemCode, legacy } = viewConfig;
     const [selectedTab, setSelectedTab] = useState(TABS.CHART);
-    const { startDate, endDate } = yearToApiDates(year);
+    const { startDate, endDate } = year ? yearToApiDates(year) : DEFAULT_DATE_RANGE;
```

Comment:
This shouldn't be needed as there is default date handling in `yearToApiDates`. If it's not working for some reason it would be best to fix it in the yearToApiDates. Note: yearToApiDates is also being used by the map overlays.

Reply:
Oh right cool! I think I tried this fix but my bug was I was sending through `year` when I shouldn't, so I can probably just remove it :-)